### ISSUE TITLE
Disabled options should not receive focus

### ISSUE THREAD

**Issue by renataogarcia** [thisbeyond/solid-select#66]
This behaviour can be observed in the "[disabled options](https://solid-select.com/?example=Disabled%2520Options)" example.
![image](https://github.com/thisbeyond/solid-select/assets/1123752/d3596b11-bf25-4c71-b5cc-1662cf6bd510)

**Comment by martinpengellyphillips** [comment]
Yep! Going to look at this next as it is long overdue being addressed 😄
